pkg/agent/client.go

```diff
@@ -458,10 +459,12 @@ func (a *Client) Serve() {
 						reason = metrics.DialFailureTimeout
 					}
 					metrics.Metrics.ObserveDialFailure(reason)
-					klog.ErrorS(err, "error dialing backend", "dialID", dialReq.Random, "connectionID", connID, "dialAddress", dialReq.Address)
+					// Do not log agent errors for remote unavailable.
+					kvs := []interface{}{"error", err, "dialID", dialReq.Random, "connectionID", connID, "dialAddress", dialReq.Address}
```

Comment:
out of curiosity,  why defining "kvs" variable is better than passing params to InfoS as is?

Reply:
You're right, simplified.

(I copied the pattern [here](https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/konnectivity-client/pkg/client/client.go#L280) but in this case there is no branching decision.)